lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -319,6 +326,10 @@ def main():
         if not task_mgr.ecs_api_supports_requirescompatibilities():
             module.fail_json(msg='botocore needs to be version 1.8.4 or higher to use launch_type')
 
+    if module.params['task_role_arn']:
+        if not module.botocore_at_least('1.10.44'):
```

Comment:
Thanks for adding this! Do you mind doing the same for launch_type above by replacing `if not task_mgr.ecs_api_supports_requirescompatibilities():` with `if not module.botocore_at_least('1.8.4'):`?